src/engine/commands.ts

```diff
@@ -647,8 +647,12 @@ export const publishCommunitiesList = addresses =>
 
 // Deletes
 
-export const publishDeletion = ({kind, address = null, id = null}) => {
-  const tags = [["k", String(kind)]]
+export const publishDeletion = ({kind = null, address = null, id = null}) => {
+  const tags = []
+
+  if (kind) {
+    tags.push(["k", String(kind)])
+  }
```

Comment:
Do you prefer to always include the kind? I didn't see this in the spec so was wondering what's the use case for including it.

Reply:
Yeah, there's an open PR: https://github.com/nostr-protocol/nips/pull/1141/files it's just so deletes can be requested more efficiently.